### PR TITLE
mention jsonapi-rb for people specifically looking for jsonapi serialization

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
Just a simple note.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 ## About
 
+ActiveModelSerializers is undergoing some rennovations.
+
+In the mean time, if you're looking for a json:api renderer, [jsonapi-rb](http://jsonapi-rb.org/) will meet your needs --
+with a [performance boost](https://gist.github.com/NullVoxPopuli/748e89ddc1732b42fdf42435d773734a), too.
+
```
Comment (bf4) [inline README.md]: Maybe instead of adding this text  here, add here a link to ```[Development Status](#status-of-ams)``` and then put this down there?

Comment (bf4) [reply README.md]: And maybe reference https://github.com/rails-api/active_model_serializers/tree/benchmarks per discussion in https://github.com/rails-api/active_model_serializers/pull/2169 ??

File: README.md [hunk 2]
```diff
@@ -41,6 +43,13 @@ Please see below for the documentation relevant to you.
 - ❗️ All existing PRs against master will need to be closed and re-opened against 0-10-stable, if so desired
 - ❗️ Master, for the moment, won't have any released version of AMS on it.
 
+In the mean time, if you're looking for a json:api renderer, [jsonapi-rb](http://jsonapi-rb.org/) will meet your needs --
+with a [performance boost](https://gist.github.com/NullVoxPopuli/748e89ddc1732b42fdf42435d773734a), too.
```
Comment (bf4) [inline README.md]: Maybe something like?


```
### Alternatives

- [jsonapi-rb](http://jsonapi-rb.org/) is a [highly performant](https://gist.github.com/NullVoxPopuli/748e89ddc1732b42fdf42435d773734a) and modular JSON:API-only implementation.  There's a vibrant community around it that has produced projects such as [JSON:API Suite](https://jsonapi-suite.github.io/jsonapi_suite/).

For benchmarks against alternatives, see https://github.com/rails-api/active_model_serializers/tree/benchmarks
```

Comment (NullVoxPopuli) [reply README.md]: works for me!

File: README.md [hunk 3]
```diff
@@ -41,6 +43,12 @@ Please see below for the documentation relevant to you.
...
 
+### Alternatives
+
+- [jsonapi-rb](http://jsonapi-rb.org/) is a [highly performant](https://gist.github.com/NullVoxPopuli/748e89ddc1732b42fdf42435d773734a) and modular JSON:API-only implementation.  There's a vibrant community around it that has produced projects such as [JSON:API Suite](https://jsonapi-suite.github.io/jsonapi_suite/).
+
+For benchmarks against alternatives, see https://github.com/rails-api/active_model_serializers/tree/benchmarks
+
 *Changes to 0.10.x maintenance*:
 
 - The 0.10.x version has become a huge maintenance version.  We had hoped to get it in shape for a 1.0 release, but it is clear that isn't going to happen.  Almost none of the maintainers from 0.8, 0.9, or earlier 0.10 are still working on AMS. We'll continue to maintain 0.10.x on the 0-10-stable branch, but maintainers won't otherwise be actively developing on it
```
Comment (bf4) [inline README.md]: s/version/burden

lolz

Comment (NullVoxPopuli) [reply README.md]: :-D

Comment (NullVoxPopuli) [reply README.md]: I should probably actually have a different subheader for this stuff

File: README.md [hunk 4]
```diff
@@ -2,6 +2,8 @@
 
 ## About
 
+ActiveModelSerializers is undergoing some renovations. see [Development Status](#status-of-ams)
```
Comment (bf4) [inline README.md]: missing `.`

File: README.md [hunk 5]
```diff
@@ -2,9 +2,7 @@
 
 ## About
 
-## Installation
-
-## Getting Started
+ActiveModelSerializers is undergoing some renovations. See [Development Status](#status-of-ams).
```
Comment (bf4) [inline README.md]: Sure

File: README.md [hunk 6]
```diff
@@ -22,31 +20,31 @@ Thanks!
...
 
 - ❗️ All existing PRs against master will need to be closed and re-opened against 0-10-stable, if so desired
 - ❗️ Master, for the moment, won't have any released version of AMS on it.
+- :eyes: See below for [alternatives](#alternatives)
 
-*Changes to 0.10.x maintenance*:
+
+### *Changes to 0.10.x maintenance*:
 
 - The 0.10.x version has become a huge maintenance version.  We had hoped to get it in shape for a 1.0 release, but it is clear that isn't going to happen.  Almost none of the maintainers from 0.8, 0.9, or earlier 0.10 are still working on AMS. We'll continue to maintain 0.10.x on the 0-10-stable branch, but maintainers won't otherwise be actively developing on it
```
Comment (bf4) [inline README.md]: would you mind fixing the typo whilst here? s/version/burden? and add a period to end of bullet line?

File: README.md [hunk 7]
```diff
@@ -22,31 +20,31 @@ Thanks!
 If you're reading this at https://github.com/rails-api/active_model_serializers you are
 reading documentation for our `master`, which is not yet released.
 
-Please see below for the documentation relevant to you.
-
 - [0.10 (0-10-stable) Documentation](https://github.com/rails-api/active_model_serializers/tree/0-10-stable)
 - [0.10.6 (latest release) Documentation](https://github.com/rails-api/active_model_serializers/tree/v0.10.6)
-  - [![API Docs](http://img.shields.io/badge/yard-docs-blue.svg)](http://www.rubydoc.info/gems/active_model_serializers/0.10.6)
```
[the remark this answers is not in the document]
Comment (NullVoxPopuli) [reply README.md]: can do

File: README.md [hunk 8]
```diff
@@ -22,31 +20,38 @@ Thanks!
...
-- [0.10.6 (latest release) Documentation](https://github.com/rails-api/active_model_serializers/tree/v0.10.6)
-  - [![API Docs](http://img.shields.io/badge/yard-docs-blue.svg)](http://www.rubydoc.info/gems/active_model_serializers/0.10.6)
-  - [Guides](https://github.com/rails-api/active_model_serializers/tree/v0.10.6/docs)
-- [0.9 (0-9-stable) Documentation](https://github.com/rails-api/active_model_serializers/tree/0-9-stable)
-  - [![API Docs](http://img.shields.io/badge/yard-docs-blue.svg)](http://www.rubydoc.info/github/rails-api/active_model_serializers/0-9-stable)
-- [0.8 (0-8-stable) Documentation](https://github.com/rails-api/active_model_serializers/tree/0-8-stable)
-  - [![API Docs](http://img.shields.io/badge/yard-docs-blue.svg)](http://www.rubydoc.info/github/rails-api/active_model_serializers/0-8-stable)
+<table>
+  <tr>  
+    <td> [0.10 (0-10-stable) Documentation](https://github.com/rails-api/active_model_serializers/tree/0-10-stable) </td>
```
Comment (NullVoxPopuli) [inline README.md]: this looks better in atom. :-\

Comment (NullVoxPopuli) [reply README.md]: 2017-09-19-104959_486x186_scrot

File: README.md [hunk 9]
```diff
@@ -22,31 +20,63 @@ Thanks!
...
-- [0.10.6 (latest release) Documentation](https://github.com/rails-api/active_model_serializers/tree/v0.10.6)
-  - [![API Docs](http://img.shields.io/badge/yard-docs-blue.svg)](http://www.rubydoc.info/gems/active_model_serializers/0.10.6)
-  - [Guides](https://github.com/rails-api/active_model_serializers/tree/v0.10.6/docs)
-- [0.9 (0-9-stable) Documentation](https://github.com/rails-api/active_model_serializers/tree/0-9-stable)
-  - [![API Docs](http://img.shields.io/badge/yard-docs-blue.svg)](http://www.rubydoc.info/github/rails-api/active_model_serializers/0-9-stable)
-- [0.8 (0-8-stable) Documentation](https://github.com/rails-api/active_model_serializers/tree/0-8-stable)
-  - [![API Docs](http://img.shields.io/badge/yard-docs-blue.svg)](http://www.rubydoc.info/github/rails-api/active_model_serializers/0-8-stable)
+<table>
+  <tr>  
+    <td>
```
Comment (NullVoxPopuli) [inline README.md]: html. blegh. but the result is nice.

**Pull request merged**